src/main.cpp

```diff
@@ -429,126 +720,299 @@ int main(int argc, char *argv[])
-		if (vramMonitorEnabled)
-		{
-			mvprintw(6, 0, "%s", fmt::format("VRAM target: {}%", std::to_string(vramTarget * 100).substr(0, 4)).c_str());
-			mvprintw(7, 0, "%s", fmt::format("VRAM limit: {}%", std::to_string(vramLimit * 100).substr(0, 4)).c_str());
-		}
-		else
-		{
-			mvprintw(6, 0, "%s", fmt::format("VRAM target: Disabled").c_str());
-			mvprintw(7, 0, "%s", fmt::format("VRAM limit: Disabled").c_str());
-		}
+			// Set position and size to fill the viewport
+			ImGui::SetWindowPos(ImVec2(0, 0));
+			ImGui::SetWindowSize(ImVec2(mainWindowWidth, mainWindowHeight));
 
-		// FPS and frametimes
-		mvprintw(9, 0, "%s", fmt::format("FPS: {} fps", std::to_string(currentFps)).c_str());
-		mvprintw(10, 0, "%s", fmt::format("GPU frametime: {} ms", std::to_string(averageGpuTime).substr(0, 4)).c_str());
-		mvprintw(11, 0, "%s", fmt::format("CPU frametime: {} ms", std::to_string(averageCpuTime).substr(0, 4)).c_str());
-		mvprintw(12, 0, "%s", fmt::format("Raw CPU frametime: {} ms", std::to_string(realCpuTime).substr(0, 4)).c_str());
+			// Title
+			ImGui::Text("OVR Dynamic Resolution");
 
-		// VRAM usage
-		if (vramMonitorEnabled)
-			mvprintw(13, 0, "%s", fmt::format("VRAM usage: {}%", std::to_string(vramUsage * 100).substr(0, 4)).c_str());
-		else
-			mvprintw(13, 0, "%s", fmt::format("VRAM usage: Disabled").c_str());
+			ImGui::Separator();
+			ImGui::NewLine();
 
-		// Reprojecting status
-		if (frameShown > 1)
-		{
-			std::string reason = fmt::format("Other [{}]", reprojectionFlag).c_str();
-			if (reprojectionFlag == 20)
-				reason = "CPU";
-			else if (reprojectionFlag == 276)
-				reason = "GPU";
+			// HMD Hz
+			ImGui::Text(fmt::format("HMD refresh rate: {} hz ({} ms)", std::to_string(hmdHz), std::to_string(hmdFrametime).substr(0, 4)).c_str());
+
+			// Target FPS and frametime
+			if (!vramOnlyMode)
+			{
+				ImGui::Text(fmt::format("Target FPS: {} fps ({} ms)", std::to_string(targetFps), std::to_string(targetFrametime).substr(0, 4)).c_str());
+			}
+			else
+			{
+				ImGui::Text("Target FPS: Disabled");
+			}
+
+			// VRAM target and limit
+			if (nvmlEnabled && nvmlEnabled)
+			{
+				ImGui::Text(fmt::format("VRAM target: {} GB", std::to_string(vramTarget / 100.0f * vramTotalGB).substr(0, 4)).c_str());
+				ImGui::Text(fmt::format("VRAM limit: {} GB ", std::to_string(vramLimit / 100.0f * vramTotalGB).substr(0, 4)).c_str());
+			}
+			else
+			{
+				ImGui::Text(fmt::format("VRAM target: Disabled").c_str());
+				ImGui::Text(fmt::format("VRAM limit: Disabled").c_str());
+			}
 
-			mvprintw(15, 0, fmt::format("Reprojecting: Yes ({}x, {})", frameShown, reason).c_str());
+			ImGui::NewLine();
+
+			// FPS and frametimes
+			ImGui::Text(fmt::format("FPS: {} fps", std::to_string(currentFps)).c_str());
+			ImGui::Text(fmt::format("GPU frametime: {} ms", std::to_string(averageGpuTime).substr(0, 4)).c_str());
+			ImGui::Text(fmt::format("CPU frametime: {} ms", std::to_string(averageCpuTime).substr(0, 4)).c_str());
```

Comment:
The call to `substr(0,4)` will not, in general, return a string of fixed width. E.g., if the string is "0", then you get "0". If the string is "10000", then I suppose you get "1000". Whether that's what you want... I am not sure.

```suggestion
			ImGui::Text(fmt::format("CPU frametime: {:10.2f} ms", averageCpuTime);
```

This will always use at least 10 characters with 2 decimal digits. Such an approach could work nicely.

Reply:
Thanks, though what I was actually searching for was `:.2f` since I don't want a minimum of characters before the decimal point (or else it's filled with spaces)

Reply:
@Erimelowo Note that you can even turn it into a parameter.

So you can write something like 

```C++
ImGui::Text(fmt::format("CPU frametime: {:.{digits}f} ms", averageCpuTime, digits);
```

See the `fmt` documentation for details. :-)